### ISSUE TITLE
Dynamic custom entity

### ISSUE THREAD

**Issue by purnadika** [cofoundry-cms/cofoundry#365]
It would better if there's a Data model annotation that can trigger which CustomEntity entries should be loaded on the next field.
`
public string SelectedCustomEntity { get; set; }

        [Display(Description = "Entity Type.")]
        [CustomEntity(SelectedCustomEntity)]
        public int EntityEntry { get; set; }`

**Comment by HeyJoel** [comment]
No, we don't have a built-in annotation for that, you'd need to add each entity as separate properties. I suppose the editor would work like the multi-type custom entity picker, where each entity type is a separate 'add' option in the menu.
